lms/templates/static_templates/ferie_bez_nudy.html

```diff
@@ -0,0 +1,549 @@
+                <button class="webinar__block__content-video col" type="button" aria-pressed="false" aria-label="Otwórz webinar Podstawy prawa karnego i cywilnego" data-toggle="modal" data-target="#modal-webinar-prawo-video-1" style="background-image: url('/static/images/webinars/podstawy-prawa-karnego-i-cywilnego.jpg');">
+                    <span class="available-soon">Dostępny wkrótce</span>
+                </button>
+                <div class="webinar__block__content-description col p-3">
+                    <span class="webinar__block__content-description--date font-weight-bold">Webinar</span>
+                    <h3 class="webinar__block__content-description--title my-1">Podstawy prawa karnego i cywilnego</h3>
+                    <p class="webinar__block__content-description--text">
+                        W trakcie wykładu zaprezentowano podstawy prawa karnego oraz prawa cywilnego. W części dotyczącej prawa karnego omówione zostały funkcje prawa karnego oraz zasady postępowania w sprawach nieletnich. Wykład z prawa cywilnego dotyczył przede wszystkim praw konsumentów oraz podstawowych zagadnień prawa rodzinnego.
+                    </p>
+                    <p class="webinar__block__content-more">
+                        <button class="webinar__block-link ff-OpenSans font-weight-bold js-toogle-more" data-toggle="collapse" data-target="#collapse-webinar-prawo-1" type="button" aria-expanded="false" aria-controls="collapse-webinar-prawo-1" aria-describedby="collapse-webinar-prawo-1">Więcej informacji <span class="sr-only">o Podstawy prawa karnego i cywilnego</span></button>
+                        <i class="fas fa-chevron-down ml-1" aria-hidden="true"></i>
+                    </p>
+                </div>
+                <div class="webinar__block__content--more-info p-3 col-12 collapse" id="collapse-webinar-prawo-1">
+                    <div class="d-flex flex-row flex-wrap mb-1">
+                        <h4 class="webinar__block__content--more-info-h4 mr-2">Prowadzący: </h4><span><strong>dr Krzysztof Szczucki</strong>, Wydział Prawa i Administracji Uniwersytetu Warszawskiego</span>
+                    </div>
+                    <div class="invisible">
+                        <h4 class="webinar__block__content--more-info-h4">Opis:</h4>
+                        <p class="webinar__block__content--more-info-descr">
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce non erat at quam rutrum pulvinar sit amet quis tellus. Sed eu congue lacus.
+                        </p>
+                    </div>
+                    <div class="invisible">
+                        <h4 class="webinar__block__content--more-info-h4">Przydatne linki:</h4>
+                        <ul class="webinar__block__content--more-info-links px-3">
+                            <li><a href="#" target="_blank" class="webinar__block-link">mockup link</a></li>
```

Comment:
braki + tekst

Reply:
to są elementy ukryte, miały być dodatkowe opisy, materiały od autorów, ale mało kto co dostarczył, zostało na ewentualne zaktualizowanie.